src/ServicePulse.Host/ServicePulse.Host.csproj

```diff
@@ -227,6 +227,8 @@
     <Content Include="app\js\directives\ui.particular.tabset.tpl.html" />
     <Content Include="app\js\views\endpoints\endpoints.controller.js" />
     <Content Include="app\js\views\endpoints\endpoints.route.js" />
+    <Content Include="app\js\views\monitored-endpoints\monitored-endpoints.controller.js" />
```

Comment:
@WilliamBZA in every other spot we use _ not - for directories. Was it intentional or accidental? I am fine with changing it in all places to - if we think it is better.